scripts/fetch_typeshed.py

```diff
@@ -69,20 +69,23 @@ def fetch_as_tarfile(
     url: str,
 ) -> tuple[tarfile.TarFile, FetchMetadata]:
     LOG.info(f"Fetch typeshed tarball from URL `{url}`...")
-    data = urllib.request.urlopen(url).read()
-    metadata = FetchMetadata(
-        url=url,
-        sha256=hashlib.sha256(data).hexdigest(),
-        timestamp=datetime.datetime.now().timestamp(),
-    )
-    return (
-        tarfile.open(
-            # Buffer the url data stream in memory so seeking in tarfile is possible
-            fileobj=io.BytesIO(data),
-            mode="r:gz",
-        ),
-        metadata,
-    )
+    try:
```

Comment:
this change doesn't seem related to the issue? same with the lint ignore comment below